target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-x1pro.dtsi

```diff
@@ -0,0 +1,224 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
Why do you have this additional dtsi file? Are there multiple variants of this device you plan to ad later?

Reply:
> Are there multiple variants of this device you plan to ad later?

Yes, a 5G version